### DISCUSSION TITLE
Hi there, is there anyone active in here ?

### DISCUSSION THREAD

**Discussion by YanX94** [GeorgianStan/vanilla-context-menu#21] [General]
Hi there, is there anyone active in here ?

**Comment by GeorgianStan** [comment]
Hello!

**Comment by LeGambit** [comment]
Hi Georgian are you still active?  I've used your vanilla context menu following your instructions and was able to get it to work with fullcalendar.io .  I was wondering if you are able to help on an issue I have with the menu?

**Comment by GeorgianStan** [comment]
Hi @LeGambit, sure I would like to! What's the issue?

**Reply by LeGambit** [reply]
Hi, firstly thanks for your context menu.  I have been looking everywhere for a solution that made sense, and yours fit the bill perfectly.  Secondly thank you for getting back to me.   As mentioned I integrated your context menu with fullcalendar so that a menu appears however I am stuck in trying to link back to the event.  So let's say I right click on an event and choose option 1, I'm trying to display back to in an alert the start date and time of the event I right clicked on. You can see my code here https://codepen.io/LeGambit/pen/ZExgPjO .  Any help you can give would be greatly appreciated.  Thanks.

**Reply by GeorgianStan** [reply]
Sure! Actually that's the update I did in the latest version of this library. You can this [file](https://github.com/GeorgianStan/vanilla-context-menu/blob/master/src/index.ts) and search for `#initialContextMenuEvent`. You can implement it in the same way. 

Basically, you need to store the event that is triggered on `contextmenu`. In your codepen, that's the variable `jsEvent` from line **56** and then you can access it it like `jsEvent.target`. 

Check this updated [codepen](https://codepen.io/GeoPablo/pen/WNzVVBO?editors=0010)

**Reply by LeGambit** [reply]
That's awesome.  You're a legend.  Works great, and thank you for sharing your codepen with me.  I'm such a newbie at jquery and just trying to expand my understanding.  One further question if you don't mind.  In the console.log  it shows the html div elements which contains the event title.  However tied to each event is an event id, usually accessible via event.id or event.title, or it's extended properties (where you can store more information).  How would I store and display that in an alert so event.start and event.title is stored and can be accessed?  If not I can acccess target.innerHTML and try to manipulate it to get the properties. I promise I'll try to make it the last question, as I'm sure I'm taking up your time.  However please understand I totally appreciate your help on this, I've been stuck on this issue for a while.  Big Thank You!

**Reply by GeorgianStan** [reply]
Sure @LeGambit  no problem. Check this [codepen](https://codepen.io/GeoPablo/pen/WNzVVBO?editors=0010)

**Reply by LeGambit** [reply]
That's awesome, and works really well now.  Thank you so much, I've learned a lot from your examples.
